fontgenerator.js

```diff
@@ -433,8 +431,9 @@ TinyFontGenerator = {
     var data = ""; 
     data += " 00 01 00 00";  // version 1
     data += " 00 00 00 00";  // italicAngle
-    data += " 00 00 00 00";  // underlinePosition
-    data += " 00 00 00 00";  // underlineThickness
+    data += " 00 00";        // underlinePosition
```

Comment:
do the font not work without these fields? The main intention is for this font to work, if that means it gets rejected by OTS, that's bad, but it only encounters a warning and still works, then that is perfectly fine (this generator is not intended to generate system fonts, so if it doesn't pass TTX parsing, that's also fine)